app/styles/tasks.css

```diff
@@ -202,6 +202,57 @@ input[type=range]:focus::-ms-fill-upper {
   margin-top: 10px;
 }
 
+/* modal styles */
+
+.overlay {
+  background: rgb(0, 0, 0, 0.7);
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  z-index: 1;
+}
+
+.modal {
+  width: 90%;
+  max-width: 650px;
+  margin: 0 auto;
+  align-self: center;
+  padding: .2rem .2rem;
```

Comment:
Is there a particular format being followed here?
.2 -----> 0.2 ?

Reply:
I will change it, thanks harshith